master/internal/rm/kubernetesrm/kubernetes_resource_manager.go

```diff
@@ -572,39 +572,7 @@ func (k *ResourceManager) createResourcePoolSummary(
 func (k *ResourceManager) fetchAvgQueuedTime(pool string) (
 	[]*jobv1.AggregateQueueStats, error,
 ) {
-	aggregates := []model.ResourceAggregates{}
```

Comment:
the database query doesn't change at all in this PR right?

Reply:
no the pr should not have any effectvie change other than 

> add missing job submission time on k8s assigned resources
reuse average stats calculation between rms